OrbitCore/Capture.cpp

```diff
@@ -143,7 +144,7 @@ void Capture::PreFunctionHooks() {
   GSelectedFunctions = GetSelectedFunctions();
 
   for (auto& func : GSelectedFunctions) {
-    uint64_t address = func->GetVirtualAddress();
+    uint64_t address = function::GetVirtualAddress(*func);
```

Comment:
Probably rename it to GetAbsoluteAddress while you are here. virtual address means a different thing in the context of elf.

Reply:
Yes, thanks, renamed to GetAbsoluteAddress